keras/preprocessing/image.py

```diff
@@ -162,7 +162,12 @@ def load_img(path, grayscale=False, target_size=None):
     else:  # Ensure 3 channel even when loaded image is grayscale
         img = img.convert('RGB')
     if target_size:
-        img = img.resize(target_size)
+        targ_width=target_size[1] #The number of columns in the target corresponds to the width
+        targ_height=target_size[0] #The number of rows in the target corresponds to the height
```

Comment:
Also style: use `target_height` / `target_width`, no point in removing two characters.